static/js/settings_org.js

```diff
@@ -637,13 +637,17 @@ exports.init_dropdown_widgets = () => {
     });
 };
 
-exports.save_organization_settings = function (data, save_button) {
+exports.save_organization_settings = function (data, save_button, url) {
```

Comment:
We should rename this function to something more generic and remove the `if (!url) {` condition which is present a few lines below and pass the url in every usage.
Ideally, we should move these types of functions to a more generic place, maybe `settings_ui`?